locale/Makefile.def

```diff
@@ -0,0 +1,67 @@
+#
+# Definition of common Makefile tasks for PO merging and MO generation.
+#
+# make all-mo (default) - generate MO files
+# make check - check translations using translate-tool
+# make tx-update - download and merge translations from Transifex
+# make clean - clean everything
+#
+
+POTFILE = $(DOMAIN).pot
+MOFILE = $(DOMAIN).mo
+POFILES = $(shell find . -name '$(DOMAIN).po')
+MOFILES = $(patsubst %.po,%.mo,$(POFILES))
+POXFILES = $(patsubst %.po,%.pox,$(POFILES))
+EDITFILES = $(patsubst %.po,%.edit.po,$(POFILES))
+TIMESTAMPFILES = $(patsubst %.po,%.po.time_stamp,$(POFILES))
+
+%.mo: %.po
+	mkdir -p $(shell dirname $@)/LC_MESSAGES
+	msgfmt -o $(shell dirname $@)/LC_MESSAGES/$(MOFILE) $<
+
+# Generate MO files from PO files
+all-mo: $(MOFILES)
```

Comment:
As this is going to be shared and we no longer don't have control over what is in the project dirs I'd suggest to add .PHONY for non-file targets. Note: Makefile style guide suggests to add `.PHONY: ` immediately before the target rather then listing all at one place.
```
.PHONY: all-mo
```